test/drawers/drawerTests.ts

```diff
@@ -22,17 +21,17 @@ class MockAnimator implements Plottable.Animator {
 }
 
 function createMockDrawer(dataset: Plottable.Dataset) {
-  var drawer = new Plottable.Drawer(dataset);
+  const drawer = new Plottable.Drawer(dataset);
   (<any> drawer)._svgElementName = "circle";
   return drawer;
 }
 
 describe("Drawers", () => {
   describe("Abstract Drawer", () => {
-    var oldTimeout: any;
-    var timings: number[] = [];
-    var svg: d3.Selection<void>;
-    var drawer: Plottable.Drawer;
+    let oldTimeout: any;
+    let timings: number[] = [];
+    let svg: d3.Selection<void>;
+    let drawer: Plottable.Drawer;
```

Comment:
But they refer to the same thing. You actually return the const and then you use it as a let